nss-day-cohort-30/bangazon-api-ambitious-albatrosses
# Added department and employee models

Fixes:
-Department and employee models are now built out

Changes proposed in this request & reasons behind organizational or architectural decisions:
- We needed them

Steps to test:
- Compare contents of classes to SQL database and make sure they match

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
- N/A
-

Commit message:
-

Link to feature ticket:
-

@ambitions-albatrosses
